server-mock/db/db.go

```diff
@@ -22,6 +22,9 @@ type Database struct {
 
 	// Internal fields
 	logger *slog.Logger
+
+	// Private ETH Wallet Key for ConstellationAdmin contract
+	ConstellationAdminPrivateKey []byte
```

Comment:
This should be the native form of the key (`*ecdsa.PrivateKey`) instead of the serialized form.

Reply:
updated